pom.xml

```diff
@@ -81,6 +81,24 @@
 			<scope>runtime</scope>
 		</dependency>
 
+        <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.10.1</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <version>5.4.0</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.13.2</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
```
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/transactionsGetTest.java:[14,22] package io.restassured does not exist
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/transactionsGetTest.java:[15,27] package io.restassured.http does not exist
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/transactionsGetTest.java:[16,31] package io.restassured.response does not exist
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/transactionsGetTest.java:[17,17] package org.junit does not exist
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/transactionsGetTest.java:[19,29] package io.restassured does not exist
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/transactionsGetTest.java:[19,1] static import only from classes and interfaces
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/transactionsGetTest.java:[20,24] cannot find symbol
  symbol:   class Assert
  location: package org.junit
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/transactionsGetTest.java:[38,6] cannot find symbol
  symbol:   class Before
  location: class com.medeiros.RoostTest.transactionsGetTest
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/transactionsGetTest.java:[49,11] cannot find symbol
  symbol:   variable RestAssured
  location: class com.medeiros.RoostTest.transactionsGetTest
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/transactionsGetTest.java:[51,17] cannot find symbol
  symbol:   class Response
  location: class com.medeiros.RoostTest.transactionsGetTest
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/transactionsGetTest.java:[51,37] cannot find symbol
  symbol:   method given()
  location: class com.medeiros.RoostTest.transactionsGetTest
```

The added dependencies were missing.

**FIX** :- Added the necessary dependencies needed for the imports.